sdk/cosmos/tests/user.rs

```diff
@@ -39,15 +39,16 @@ async fn users() {
         .await
         .unwrap();
 
+    let list_users_response = database_client.list_users().execute().await.unwrap();
+    assert_eq!(list_users_response.users.len(), 0);
+
     let user_client = database_client.into_user_client(USER_NAME_REPLACED);
 
     let _delete_user_response = user_client.delete_user().execute().await.unwrap();
 
-    let list_users_response = database_client.list_users().execute().await.unwrap();
-    assert_eq!(list_users_response.users.len(), 0);
-
```

Comment:
I would have kept this, it checks if the delete has been successful.

Reply:
Whoops! Great catch!